app/com/arpnetworking/metrics/portal/reports/impl/chrome/grafana/BaseGrafanaScreenshotRenderer.java

```diff
@@ -79,9 +81,18 @@ public URI getUri(final GrafanaReportPanelReportSource source) {
         return devToolsService.nowOrOnEvent(
                 "reportrendered",
                 () -> {
-                    final Object html = devToolsService.evaluate(
-                            "document.getElementsByClassName('rendered-markdown-container')[0].srcdoc"
-                    );
+                    final Object html;
+                    try {
+                        // TODO(spencerpearson): get rid of this .get(). It's not a big deal, since
```

Comment:
Unless you are planning on addressing this in a follow-up PR, please be sure to file an issue against the project for this.

Reply:
Done: #275